website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="[RT].Toggle" @type="yielded component">
+    The `RichTooltip::Toggle` component (see below).
+  </C.Property>
+  <C.Property @name="[RT].Bubble" @type="yielded component">
+    The `RichTooltip::Bubble` component (see below).
+  </C.Property>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the tooltip should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the tooltip can't be dismissed via `esc` or "click outside" until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="true" @values={{array "true" "false"}}>
+    Assigns "soft" event listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle to control the visibility of the tooltip.
+  </C.Property>
```

Comment:
Question: any rationale for this term? I went through the categories of JavaScript events and didn't find it in there, so I'm cautiously assuming that we're defining this ourselves.

Reply:
Yes.